Product/Production/Common/CONNECTCoreLib/src/test/java/gov/hhs/fha/nhinc/connectmgr/BaseConnctionManagerCache.java

```diff
@@ -51,6 +53,8 @@
     protected static String QUERY_FOR_DOCUMENTS_DEFERRED_URL = "https://localhost:8181/QueryForDocumentsDeferredRequest";
     protected static String QUERY_FOR_DOCUMENTS_URL_22 = "https://server2:8181/QueryForDocuments";
```

Comment:
QUERY_FOR_DOCUMENTS_URL_22 looks like an unused constant. If yes, can we remove it.